Allow numbers at the beginning of user names in pmvarrun.
Note that this patch is Debian-specific; other distributions might not
support this.

Index: libpam-mount-2.14~git2+ad53f3559/src/pmvarrun.c
===================================================================
--- libpam-mount-2.14~git2+ad53f3559.orig/src/pmvarrun.c	2012-08-09 11:54:50.483436663 +0200
+++ libpam-mount-2.14~git2+ad53f3559/src/pmvarrun.c	2012-08-09 11:58:55.258022146 +0200
@@ -117,7 +117,7 @@ static bool valid_username(const char *n
 	 * more characters.
 	 */
 	if (!((*n >= 'A' && *n <= 'Z') || (*n >= 'a' && *n <= 'z') ||
-	    *n == '_'))
+	    (*n >= '0' && *n <= '9') || *n == '_'))
 		return false;
 
 	while (*n != '\0') {
